.github/workflows/node.js.yml

```diff
@@ -0,0 +1,107 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - jp-gha-conversion
+      - staging
+      # - master
+  pull_request:
+    branches:
+      - jp-gha-conversion
+      - staging
+      # - master
+  workflow_dispatch:
+
+jobs:
+  PreDeployTesting:
```

Comment:
I would name this job `build`, if you want to give it a user readable name you can use the `name` property